docs/deployment.md

```diff
@@ -1,48 +1,67 @@
 # Deployment
 
-*This section is under construction.*
+The main Jaeger backend components are released as Docker images on Docket Hub:
 
-We will soon provide Docker images for individual Jaeger components
-[Issue #157](https://github.com/uber/jaeger/pull/157),
-as well as orchestration templates, for example to run Jaeger on
-[Kubernetes](https://github.com/jaegertracing/jaeger-kubernetes)
-and [OpenShift](https://github.com/jaegertracing/jaeger-openshift).
+Component             | Repository
+--------------------- | ---
+**jaeger-agent**      | [https://hub.docker.com/r/jaegertracing/jaeger-agent/](https://hub.docker.com/r/jaegertracing/jaeger-agent/)
```

Comment:
We could remove `https://` from the text we display for the link to reduce clutter